js/resources_test.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import { test, testPerm, assert, assertEqual } from "./test_util.ts";
+import * as deno from "deno";
+
+test(function stdioResources() {
```

Comment:
prefix all of the test names with `resources`.... so call this one `resourcesStdio`

The reason for this is to easily filter tests by module name.